### PR TITLE
BaSP-TG-10: Employee table - Resources list

### PR DESCRIPTION
**Pull request by mgriolfo-g**
Table showing a list of resources of the Employee entity from the related alfon-b database.

Options available:
- Edit or create an item (through an edit or add form).
- Delete an item (from main screen).

Modal prompts show success and error messages. CSS designs made cooperatively with Juancapp and TomasBettini!

### CODE REVIEW

File: src/Components/Employees/Modal/modal.module.css [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+  .modal__display__none {
+    display: none;
+  }
+
+  .screen {
+    width: 100vw;
+    height: 100vh;
+    display: flex;
+    position: absolute;
```
Comment (leimeter-joaquin) [inline src/Components/Employees/Modal/modal.module.css:20]: position: fixed could work better, but have it in mind for the shared modal

**Review by leimeter-joaquin — APPROVED** [review]

**Review by Alfondalix — APPROVED** [review]
Good work

File: src/Components/Employees/Modal/modal.module.css (hunk 1 above)
Comment (mgriolfo-g) [reply src/Components/Employees/Modal/modal.module.css:20]: comment shared with @TomasBettini yesterday, he will be in charge of the modal design!

**Review by Alfondalix — APPROVED** [review]

**Review by amilcarHernandez24 — APPROVED** [review]

**Comment by amilcarHernandez24** [comment]
Good job

**Pull request merged**